hips/draw/simple.py

```diff
@@ -1,5 +1,156 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
+import os
+from pathlib import Path
+
+import healpy as hp
+import numpy as np
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+from skimage import transform as tf
+from typing import List
+
+from ..tiles import HipsSurveyProperties, HipsTile, HipsTileMeta
+from ..utils import WCSGeometry, compute_healpix_pixel_indices, boundaries
 
 __all__ = [
+    'draw_sky_image',
+    'make_sky_image',
 ]
+
+
+def draw_sky_image(geometry: WCSGeometry, tiles: List[HipsTile]) -> np.ndarray:
+    """Draw sky image using the simple and quick method.
+
+    Parameters
+    ----------
+    geometry : 'WCSGeometry'
+        An object of WCSGeometry
+    tiles : List[HipsTile]
+        A list of HipsTile
+
+    Returns
+    -------
+    np.ndarray
+        Returns a numpy array containing all HiPS tiles projected onto it
+    """
+    all_sky = np.zeros(geometry.shape)
+    for tile in tiles:
+        draw_tile = DrawSkyImageOneTile(geometry, tile, all_sky.shape)
+        draw_tile.compute_corners()
+        draw_tile.apply_projection()
+        all_sky += draw_tile.warp_image()
+    return all_sky
+
+
+class DrawSkyImageOneTile:
+    """A private function for drawing a tile over a sky image.
```

Comment:
"a private function" -> this is not a function, and not private. Put a better one-line summary.

Also, please link to https://hips.readthedocs.io/en/latest/drawing_algo.html#naive-algorithm and say that this class implements the method described there.